src/contracts/instances/RwaATokenInstance.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import {RwaAToken} from 'src/contracts/protocol/tokenization/RwaAToken.sol';
+import {IPool, IAaveIncentivesController, IInitializableAToken, Errors, VersionedInitializable} from 'src/contracts/protocol/tokenization/AToken.sol';
+
+contract RwaATokenInstance is RwaAToken {
+  uint256 public constant RWA_ATOKEN_REVISION = 1;
+
+  constructor(IPool pool) RwaAToken(pool) {}
+
+  /// @inheritdoc VersionedInitializable
+  function getRevision() internal pure virtual override returns (uint256) {
+    return RWA_ATOKEN_REVISION;
```

Comment:
```suggestion
    return ATOKEN_REVISION;
```
I suggest to align with `ATokenInstance`, just in case this is used by existing tooling

Reply:
[07fdcf5](https://github.com/aave/aave-v3-horizon/pull/1/commits/07fdcf5b78c2ef8d76b89c5c508e8a8410d6bd0c)